src/command_modules/azure-cli-acr/azure/cli/command_modules/acr/task.py

```diff
@@ -663,6 +786,28 @@ def _get_list_runs_message(base_message, task_name=None, image=None):
     return "{}.".format(base_message)
 
 
+def _build_identities_info(cmd, identities, is_remove=None):
```

Comment:
nit: the way `is_remove` is used, it could just be a bool with default value

Reply:
I had some pylint trouble with that earlier but it works now. See fix

Reply:
I meant if we can just have `is_remove=False` and if `is_remove:`